### PR TITLE
[FX] Remove unused code from FX decoder

### PR DESCRIPTION
**Pull request by mvafin**
### Details:
 - *Remove unused code from FX decoder*

### Tickets:
 - *ticket-id*

### CODE REVIEW

File: src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py [hunk 1]
```diff
@@ -284,8 +287,7 @@ def get_op_type(self):
             return 'UNKNOWN_TYPE_' + str(self.pt_module.op)
 
     def get_schema(self):
-        return ''
-        return self.pt_module.schema()
+        return 'NONE'
```
Comment (rkazants) [inline src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py:290]: strange function:)

Comment (mvafin) [reply src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py:290]: It is useful in TS decoder, for FX it doesn't have meaning

Comment (rkazants) [reply src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py:290]: what about standard `None` value?

Comment (mvafin) [reply src/bindings/python/src/openvino/frontend/pytorch/fx_decoder.py:290]: We need string since that is function that is used from c++ code

**Pull request merged**